WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -438,6 +443,10 @@ def execute( self, arguments ):
         self.__report( 'Completed', 'Application Finished Successfully', sendFlag = True )
       elif not watchdog.checkError:
         self.__report( 'Completed', 'Application Finished With Errors', sendFlag = True )
+        if status in (DErrno.EWMSRESC, DErrno.EWMSRESC & 255): # the status will be truncated to 0xDE (222)
```

Comment:
I think the status could never be DErrno.EWMSRESC here, so the exact match to the second value is enough